README.md

```diff
@@ -42,9 +42,9 @@ Create a new app at [apps.dev.microsoft.com](https://apps.dev.microsoft.com), or
 
 - Use an identity that will be known by the tenant you intend to use with the application
 - Copy down the **Application ID** assigned to your app, you'll need it soon.
+- Generate an **Application Secret** of the type **Password**, and copy it for later.  Note that in production apps you should always use certificates as your application secrets, but this sample will only use a shared secret password.
```

Comment:
Good catch. Thanks! I fixed this order in another V2 sample, but forgot to revisit this one.